backend/src/main/java/de/silvia/backend/security/models/User.java

```diff
@@ -32,6 +32,9 @@
     boolean accountNonLocked;
     boolean credentialsNonExpired;
 
+    public User(String username, String password) {
+    }
+
```

Comment:
Ich würde keine Methode einbauen nur um die in den Tests benutzten zu können. Zudem ist es gar nicht gut weitere Konsturktoren hinzuzufügen, die nicht funktionieren, mit dem Risiko das man die später ausversehen benutzt.

Reply:
Hab ich rausgenommen und den Test angepasst. Guter Hinweis, Danke!